java-checks-testkit/src/main/java/org/sonar/java/checks/verifier/CheckVerifier.java

```diff
@@ -239,7 +239,9 @@ private static RemediationFunction remediationFunction(AnalyzerMessage issue) {
       }
     } catch (IOException | JsonParseException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)